content/blog/helm-summit-eu-2019.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Helm Summit EU 2019"
+slug: "helm-summit-eu-2019"
+aliases: ["/blog/2019-04-18-helm-summit-eu-2019/", "/helm-summit-eu-2019/"]
```

Comment:
Ah, you don't need "/helm-summit-eu-2019/". You see this in some old blog posts because they were moved from a former naming scheme that didn't include "/blog".